server/user_account/views.py

```diff
@@ -0,0 +1,71 @@
+from django.db.models import signals
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.authtoken.models import Token
+from rest_framework import viewsets
+from .models import CustomUser
+from .serializers import UserSerializer, LoginSerializer
+from rest_framework import generics
+from django.contrib import auth
+from django.contrib.auth.hashers import check_password
+
+
+class RegistrationView(viewsets.ModelViewSet):
+    queryset = CustomUser.objects.all()
+    serializer_class = UserSerializer
+
+    def create(self, request, *args, **kwargs):
```

Comment:
Can you write a small comment explaining this method?

Reply:
Yes, I will.